crates/glaredb/src/args/local.rs

```diff
@@ -58,17 +58,13 @@ pub struct LocalClientOpts {
     #[clap(long)]
     pub max_rows: Option<usize>,
 
-    /// CA Certificate against which to verify the server’s TLS certificate.
-    #[clap(long, value_parser, default_value = "/etc/certs/ca.pem", hide = true)]
-    pub ca_cert_path: Option<String>,
-
-    /// Domain name against which to verify the server’s TLS certificate.
-    #[clap(long, value_parser, default_value = "glaredb.com", hide = true)]
-    pub domain: Option<String>,
-
-    /// Path to CA certificate for rpcsrv proxy TLS (required for TLS protocol).
-    #[clap(long, default_value = "true", hide = true)]
-    pub disable_tls: bool,
+    /// Enable RPC TLS
+    ///
+    /// (Internal)
+    ///
+    /// In the future, we'll swap this for disable_tls
+    #[clap(long, hide = true)]
+    pub enable_tls: bool,
```

Comment:
I would probably be inclined to make this (and below) still be disable_tls, but have it default to true, so that you can enable it by doing `--disable_tls=false` (which is I think the state that it was in, and what made it opt-in before?)

Reply:
Yes good point: context for this change was that when I set the default value, I couldn't override it.

I'll show you what I mean:

```shell
$ cargo run --bin glaredb -- --enable-tls=false
   Compiling sqlexec v0.5.0 (/home/greyb/Repos/glaredb/crates/sqlexec)
   Compiling pgsrv v0.5.0 (/home/greyb/Repos/glaredb/crates/pgsrv)
   Compiling rpcsrv v0.5.0 (/home/greyb/Repos/glaredb/crates/rpcsrv)
   Compiling glaredb v0.5.0 (/home/greyb/Repos/glaredb/crates/glaredb)
    Finished dev [unoptimized + debuginfo] target(s) in 4m 57s
     Running `target/debug/glaredb --enable-tls=false`
error: unexpected value 'false' for '--enable-tls' found; no more were expected

Usage: glaredb --enable-tls

For more information, try '--help'
```

Reply:
I might be able to get this: https://github.com/clap-rs/clap/issues/1649#issuecomment-1156638220